libs/components/forms/src/lib/modules/input-box/input-box.component.spec.ts

```diff
@@ -512,6 +512,10 @@ describe('Input box component', () => {
       insetIconWrapperEl.click();
 
       expect(spy).toHaveBeenCalledTimes(1);
+      if (!document.activeElement) {
+        fail('Expected there to be an active element');
```

Comment:
Can we not write `expect(el).toEqual(document?.activeElement);` instead?

Reply:
Nope. Because `el` is defined it will not let a type of `HTMLElement | undefined` to be passed into the `toEqual`.

Reply:
I fixed this as I noticed that `el` should have been `| null`.